simespresso/io_QE/tests/test_start_espresso.py

```diff
@@ -0,0 +1,24 @@
+
+import unittest
+import os
+from simespresso.io_QE import espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
```

Comment:
Maybe it would be best to have one test file per tested module as suggested in  https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/Development_guide  (```Each module file foo.py in a package has got a corresponding test module file called test_foo.py.``` )

For example,  `test_read_qe_input.py`, `test_read_qe_output.py`, `test_start_espresso.py`, and `test_start_espresso.py` should probably be combined in one file:  `test_espresso_class.py`

One could additionally or alternatively also separate some of the functionality out of `espresso_class.py`.  For example, `start_qe` would be a good candidate.